src/components/ReportActionItem/MoneyRequestAction.js

```diff
@@ -15,6 +15,10 @@ import Navigation from '../../libs/Navigation/Navigation';
 import ROUTES from '../../ROUTES';
 import styles from '../../styles/styles';
 import * as IOUUtils from '../../libs/IOUUtils';
+import {getThreadForReportActionID, buildOptimisticChatReport} from '../../libs/ReportUtils';
```

Comment:
We should import the library and then call each specific method, e.g. `ReportUtils.getThreadForReportActionID`

```suggestion
import * as ReportUtils from '../../libs/ReportUtils';
```